### DISCUSSION TITLE
Error   Checkout PRO

### DISCUSSION THREAD

**Discussion by edsongomes1** [mercadopago/sdk-js#11] [Q&A] [answered]
Thanks for examples. I had a problem when tested Checkout PRO example.  Can't open mercadopago window.  I used localhost

<!DOCTYPE html>
<html>
<body>
    <div class="cho-container"></div>
    <script src="https://sdk.mercadopago.com/js/v2"></script>
    <script>
        const mp = new MercadoPago('PUBLIC_KEY', {
            locale: 'en-US'
        })
        const checkout = mp.checkout({
            preference: {
                id: 'YOUR_PREFERENCE_ID'
            }
        });
        checkout.render({
            container: '.cho-container',
            label: 'Pay'
        });
    </script>
</body>
</html>

Thanks!!

**Comment by estefaniadiazartucio** [comment, marked as answer]
Hi @edsongomes1 ! 

Please make sure you have configured your credentials and `preferenceId` accordingly, as well as check that you have an existing element that can serve as a container for the checkout's payment button with a className that matches the one you are passing as a param to the `render` object (in this case it should be "**cho-container**"). 

If you have any more questions or issues with the integration feel free to reply to this with more information on the matter.

**Comment by penriquehermosilla** [comment]
how can you configure your preferenceId? I didn't find anything about that

**Reply by estefaniadiazartucio** [reply]
Hi @penriquehermosilla! 
You can find how to create your preference and what you need for your integration in [the official documentation site](https://www.mercadopago.com.ar/developers/en/docs/checkout-pro/integrate-checkout-pro) which explains everything step by step. 

If you want to setup additional configurations for the preference or check all the customizations you can make you can do so [in this other documentation page](https://www.mercadopago.com.ar/developers/en/docs/checkout-pro/checkout-customization/preferences).
